inbm/dispatcher-agent/fpm-template/usr/share/dispatcher-agent/node_update_schema.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
add this file to both Yocto and default AppArmor profiles

Reply:
It should be included in the existing profile, since it says this already: /usr/share/dispatcher-agent/*.json r,